<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pageLicense</class>
 <widget class="QWidget" name="pageLicense">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/lge_perplexity2_24x24.png</normaloff>:/icons/lge_perplexity2_24x24.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblHeader">
     <property name="text">
      <string>[SettingsPageLicense.lblHeader]</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="lblStatusTitle">
       <property name="text">
        <string>[SettingsPageLicense.lblStatusTitle]</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lblStatusValue">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblEditionTitle">
       <property name="text">
        <string>[SettingsPageLicense.lblEditionTitle]</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="lblEditionValue">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblDaysTitle">
       <property name="text">
        <string>[SettingsPageLicense.lblDaysTitle]</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="lblDaysValue">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblMachineTitle">
       <property name="text">
        <string>[SettingsPageLicense.lblMachineTitle]</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="lblMachineValue">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblSourceTitle">
       <property name="text">
        <string>[SettingsPageLicense.lblSourceTitle]</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="lblSourceValue">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lblActivatedTitle">
       <property name="text">
        <string>[SettingsPageLicense.lblActivatedTitle]</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="lblActivatedValue">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="lblKeyTitle">
       <property name="text">
        <string>[SettingsPageLicense.lblKeyTitle]</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QPlainTextEdit" name="editLicenseKey">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>90</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
       </property>
       <property name="placeholderText">
        <string>[SettingsPageLicense.editLicenseKey.placeholder]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnActivate">
       <property name="text">
        <string>[SettingsPageLicense.btnActivate]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCopyDiag">
       <property name="text">
        <string>[SettingsPageLicense.btnCopyDiag]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
